sway/desktop/fx_renderer/fx_renderer.c

```diff
@@ -930,7 +930,9 @@ void fx_render_blur_effects(struct fx_renderer *renderer, const float matrix[sta
 		float blur_contrast, float blur_saturation) {
 	struct effects_shader shader = renderer->shaders.blur_effects;
 
-	glEnable(GL_BLEND);
+	glDisable(GL_BLEND);
+	glDisable(GL_STENCIL_TEST);
```

Comment:
Why do we disable the stencil? Shouldn't we disable it in whatever function enables it?

Reply:
Oh yeah! My bad. I was only intending to push the removal of the glEnable line, not the 2 others